src/components/calcite-label/calcite-label.scss

```diff
@@ -13,71 +13,69 @@
 }
 
 :host([scale="s"]) {
-  --calcite-label-margin-bottom: 12px;
+  --calcite-label-margin-bottom: theme("spacing.3");
   label {
     @apply text--2;
   }
 }
 :host([scale="m"]) {
-  --calcite-label-margin-bottom: 14px;
+  --calcite-label-margin-bottom: theme("spacing[3.5]");
   label {
     @apply text--1;
   }
 }
 :host([scale="l"]) {
-  --calcite-label-margin-bottom: 18px;
+  --calcite-label-margin-bottom: theme("spacing.4");
   label {
     @apply text-1;
   }
 }
 
 :host label {
-  cursor: pointer;
-  width: 100%;
-  margin: 0 0 var(--calcite-label-margin-bottom, $baseline) 0;
-  line-height: 1.3;
+  @apply cursor-pointer 
+    w-full 
+    mt-0 
+    mr-0 
+    ml-0
+    leading-snug;
+  margin-bottom: var(--calcite-label-margin-bottom, $baseline);
```

Comment:
fallback value is weird here. I would just set that on host if you always want a fallback. Trying to get rid of the random SCSS vars. You won't need this line at all if you axe the CSS var all together.

Reply:
@paulcpederson, I'm unsure how to proceed with this one.

Reply:
Just remove the fallback I think.

Reply:
GONE!